resources/src/main/res/values/strings.xml

```diff
@@ -87,6 +87,8 @@
   <string name="msg_opened_project_does_not_exist">Last opened project doesn\'t exist!</string>
   <string name="msg_create_new_project">Create new project</string>
   <string name="msg_open_existing_project">Open existing project</string>
+  <string name="msg_project_list">Project list</string>
+  <string name="msg_recently_opened_projects">Recently opened projects</string>
```

Comment:
Change this to `Recent projects` so that the text is not truncated on most devices.